package day42_Exceptions;

public class ThrowsKeyword2 {
    public static void main(String[] args) {

        System.out.println("Test Started");
        System.out.println(8/0);
        System.out.println("Test Completed");
        

    }
}
